02-Git/homework.c

```diff
@@ -0,0 +1,5 @@
+#include <stdio.h>
+
+int main(){

```

Comment:
Formatting issue: no space between () and {. Also please check if kernel style require opening bracket on the new line.
Make "Add Makefile and skeleton of the program" this commit title.
Add prefix to all commit titles, e.g: 02-Git: Add Makefile and skeleton of the program